app/views/pages/privacy.html.erb

```diff
@@ -4,22 +4,77 @@
+    <p>Please contact us at <a href="mailto:support@placecal.org">support@placecal.org</a> if you wish to make a request.</p>
+    
+    <h4>How to complain</h4>
+    <p>If you have any concerns about our use of your personal information, you can make a complaint to us at support@placecal.org</p>
+    <p>You can also complain to the ICO if you are unhappy with how we have used your data.</p>
+    <p>The ICO’s address:</p>
+    <p>Information Commissioner’s Office</p>
+    <p>Wycliffe House</p>
+    <p>Water Lane</p>
+    <p>Wilmslow</p>
+    <p>Cheshire</p>
+    <p>SK9 5AF</p>
+    <p>Helpline number: <a href="tel:0303-123-1113">0303 123 1113</a></p>
+    <p>ICO website: <a href="https://www.ico.org.uk" target="_blank">https://www.ico.org.uk</a></p>
 
-      <h3>Children’s Information</h3>
-      <p>Another part of our priority is adding protection for children while using the internet. We encourage parents and guardians to observe, participate in, and/or monitor and guide their online activity.</p><p>PlaceCal.org does not knowingly collect any Personal Identifiable Information from children under the age of 13. If you think that your child provided this kind of information on our website, we strongly encourage you to contact us immediately and we will do our best efforts to promptly remove such information from our records.</p>
+    <h3>Web Tracking</h3>
+    
+    <h4>Log Files</h4>
+    <p>PlaceCal.org follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analysing trends, administering the site, tracking users’ movement on the website, and gathering demographic information.</p>
 
-      <h3>Online Privacy Policy Only</h3>
-      <p>This privacy policy applies only to our online activities and is valid for visitors to our website with regards to the information that they shared and/or collect in PlaceCal.org. This policy is not applicable to any information collected offline or via channels other than this website.</p>
+    <h4>Cookies, Web Beacons and Third Party Privacy Policies</h4>
+
+    <p>PlaceCal doesn’t store first party cookies.</p>
+
+    <p>PlaceCal uses a third party service called Plausible which allows us to track site visits. You can consult Plausible’s Privacy Policy here: <a href="https://plausible.io/data-policy" target="_blank">https://plausible.io/data-policy</a>. Plausible does not use cookies to track your data, nor does it store any personal data or personally identifiable information (PII).</p>
```

Comment:
We should maybe move this up under the Log files section - or repurpose that section to incorporate this instead?

Reply:
sounds like busywork to me :)